macos/src/vlc/libvlc.h

```diff
@@ -0,0 +1,580 @@
+/*****************************************************************************
```

Comment:
It would be really cool, if we could avoid storing headers of libVLC & libVLC++ separately and use existing archives at ./bin. I don't mind changing location of those files if you wish.

Reply:
Of course. I think that it would be better if dartvlc is seperated as a library(static and shared), as mentioned by jnschulze. And maybe moving both `dartvlc/*` and `ffi/native*`(and `./bin`) into `dartvlc/` is better.